app/scripts/modules/core/src/loadBalancer/loadBalancer.read.service.ts

```diff
@@ -5,14 +5,15 @@ import { INFRASTRUCTURE_CACHE_SERVICE, InfrastructureCacheService } from 'core/c
 import { NAMING_SERVICE, NamingService, IComponentName } from 'core/naming/naming.service';
 import { ILoadBalancer, ILoadBalancerSourceData } from 'core/domain';
 
-export interface ILoadBalancersByRegion {
-  name: string;
-  loadBalancers: ILoadBalancerSourceData[];
-}
-
 export interface ILoadBalancersByAccount {
   name: string;
-  accounts: ILoadBalancersByRegion[];
```

Comment:
Not related to this PR, but this shape was wrong. Also collapsed into one interface since the others are never used.